selfdrive/ui/ui.cc

```diff
@@ -300,7 +307,8 @@ void Device::setAwake(bool on) {
 }
 
 void Device::resetInteractiveTimout() {
```

Comment:
this is a net +12 lines for a super simple change. I think it would be cleaner to be able to call `Device::resetInteractiveTimout` from the driver view code and pass it a timeout. this way it's more generic and the Device class doesn't need to know anything about driver view